lib/hashie/extensions/dash/indifferent_access.rb

```diff
@@ -11,6 +11,7 @@ module ClassMethods
           # Check to see if the specified property has already been
           # defined.
           def property?(name)
+            name = defined?(translations) && translation_exists?(name) ? translations[:"#{name}"] : name
```

Comment:
specs still pass with the change. Do you have a preferred way to "check whether we have translations"? I could do ```ruby
included_modules.include?(Hashie::Extensions::Dash::PropertyTranslation)```

Not sure that's any prettier/better but if it's preferred I can do that